modules/099-ceph-csi/tools/rbd-in-tree-to-ceph-csi-migration-helper.sh

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 Flant JSC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [ "$#" != "2" ] || ! grep -qs "/" <<< "$1" || ! grep -qs "/" <<< "$2"; then
+  echo "Not enough arguments passed or namespace is not specified."
+  echo "Usage: ./rbd-in-tree-to-ceph-csi-migration-helper.sh <namespace>/<sample-pvc-name> <namespace>/<target-pvc-name>"
```

Comment:
better `<namespace>/<source-pvc-name>`instead of `<namespace>/<sample-pvc-name>`